libcxx/include/__bit/countl.h

```diff
@@ -25,15 +25,27 @@ _LIBCPP_PUSH_MACROS
 _LIBCPP_BEGIN_NAMESPACE_STD
 
 _LIBCPP_NODISCARD inline _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR int __libcpp_clz(unsigned __x) _NOEXCEPT {
+#if __has_builtin(__builtin_clzg)
```

Comment:
Please add comment at the top of the file with a TODO explaining that `__builtin_clzg` is available with
starting clang-19 and gcc-14. That should help is to know when we can enable the new builtins unconditionally.

Reply:
Done. To all three files.